include/cutlass/epilogue/collective/builders/pvc_builder.inl

```diff
@@ -0,0 +1,130 @@
+ * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ *
+ **************************************************************************************************/
+
+#pragma once
+
+#include <cutlass/arch/arch.h>
+
+#include "cutlass/epilogue/collective/default_epilogue.hpp"
+#include "cutlass/epilogue/collective/intel_pvc_epilogue.hpp"
+#include "cutlass/epilogue/fusion/intel_pvc_callbacks.hpp"
+
+
+namespace cutlass::epilogue::collective {
+
+  // PVC epilogue builder
+
+template <
+  class TileShape_MNK,
+  class EpilogueTileType,
+  class ElementAccumulator,
+  class ElementCompute,
+  class ElementC_,
+  class GmemLayoutTagC_,
+  int AlignmentC,
+  class ElementD,
+  class GmemLayoutTagD,
+  int AlignmentD,
+  class FusionOpOrCallbacks
+  >
+  class CollectiveBuilder<
+      arch::IntelPVC,
+      arch::OpClassTensorOp, 
+      TileShape_MNK,
+      Shape<_1, _1, _1>,    // Cluster Shape, which has no meaning in PVC
+      EpilogueTileType,
+      ElementAccumulator,
+      ElementCompute,
+      ElementC,
+      GmemLayoutTagC,
+      AlignmentC,
+      ElementD,
+      GmemLayoutTagD,
+      AlignmentD,
+      EpilogueScheduleAuto, // We do not have different type of epilogue support yet
+      FusionOpOrCallbacks,
+      cute::enable_if_t<
+        cute::is_same_v<GmemLayoutTagC,  cutlass::layout::RowMajor> &&
+        cute::is_same_v<GmemLayoutTagD,  cutlass::layout::RowMajor> &&
+        cute::is_same_v<EpilogueTileType, EpilogueTileAuto> &&
+        // Only linear combination is supported at the moment
+        cute::is_same_v<FusionOpOrCallbacks, 
+                        cutlass::epilogue::fusion::LinearCombination<ElementD,ElementCompute,ElementC,ElementCompute>>
+      >
+    >{
+      #ifndef SYCL_INTEL_TARGET
+        static_assert(cutlass::detail::dependent_false<arch::IntelPVC>, 
+                      "Trying to use Intel PVC pipeline when target device is not intel_gpu_pvc")
+      #endif
+      static_assert(is_static<TileShape_MNK>::value);
+      static_assert(cute::is_same_v<ElementC, float>, "ElementC needs to be float for PVC pipeline");
+      
+      // This is not strictly required ?
+      //static_assert(cute::is_same_v<ElementC, ElementD>, "ElementC and ElementD needs to be of the same type");
+      
+      // PVC epilogue with linear combination does not impose any alignment restrictions on C or D.
+      (void) AlignmentC;
+      (void) AlignmentD;
+
+      using TiledMma = TiledMMA<MMA_Atom<XE_8x16x16_F32BF16BF16F32_TT>,
+              Layout<Shape<_1,_1,_1>>,
+              Tile<_32,_64,_32>>;  // Subgroup level-tile
+      
+      using DispatchPolicy = cutlass::epilogue::IntelPVCEpilogue;
+      using CopyOpG2R = XE_2D_U32x8x16x1x1_LD_N;
+      using CopyOpR2G = XE_2D_U32x8x16x1x1_ST_N;
+
+      // PVC Epilogue with Linear Combination does not use shared memory
```

Comment:
I went ahead with explicitly stating linear combination because, 
1. This struct is specialized for the linear combination epilogue
2. For all we know, some other epilogue might